core/rest-client/src/http_client.rs

```diff
@@ -444,54 +444,92 @@ mod tests {
 	}
 
 	#[test]
+	//This test run because the request throws an error "Resource temporarily unavailable" and not because the
+	//certificate is blocked. This occurs when there is a timeout
 	fn get_from_site_with_letsencrypt_isrgrootx1_valid_certificate_fails() {
 		let base_url = Url::parse("https://valid-isrgrootx1.letsencrypt.org").unwrap();
 		let result = send_http_get_request(base_url);
+		println!("letsencrypt_isrgrootx1_valid error {:?}: ", result);
 		assert_matches!(result, Err(Error::HttpReqError(_)));
 	}
 
 	#[test]
+	//This test run because the request throws an error "Resource temporarily unavailable" and not because the
```

Comment:
^^^ 


```suggestion
	// FIXME: This test passes because the request throws an error "Resource temporarily unavailable" due to a timeout error.
	// Technically, it should check for a blocked, invalid `letsencrypt_isrgrootx1` certificate.
```